lib/diagnostic.rb

```diff
@@ -91,13 +91,13 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = 'arr.slice[-1 -2]'
```

Comment:
this results in `[12, 34, 56, 67]`

It should remove the last two indices. 

`arr.pop(2)`